test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SigningTestFixture.cs

```diff
@@ -50,6 +50,19 @@ public TrustedTestCert<TestCertificate> TrustedTestCertificate
             }
         }
 
+        public TrustedTestCert<TestCertificate> TrustedTestCertificate2
```

Comment:
Why have 2 trusted certs? I don't like that the names are not descriptive, I would opt to generate them inside the class they are used or giving them a better name here. For example `TrustedTestAuthorCertificate` and `TrustedTestRepositoryCertificate`, even if virtually they are the same I like the idea of having names that tell you what they exist for.